frontend/view-edit-client/interactions/add-client-interaction.component.tsx

```diff
@@ -20,7 +23,15 @@ export default function AddClientInteraction(props: AddClientInteractionProps) {
   const firstInputRef = React.useRef(null);
   const [servicesResponse, setServicesResponse] = React.useState(null);
   const [tempInteractionIds, setTempInteractionIds] = React.useState([0]);
+  const [interactionGetters, setInteractionGetters] = React.useState<
```

Comment:
React.useState() instead of React.useState<>

Reply:
`React.useState<Array<InteractionGetter>>([])` is valid typescript. See [Typescript generics](https://www.typescriptlang.org/docs/handbook/generics.html)